clubs/serializers.py

```diff
@@ -493,6 +496,15 @@ def get_file_url(self, obj):
         else:
             return self.context['request'].build_absolute_uri(obj.file.url)
 
+    # Cannot exceed maximum upload size
+    def validate_file(self, data):
+        if data.size <= settings.MAX_FILE_SIZE:
+            return data
+        else:
+            raise serializers.ValidationError('You cannot upload a file that is more than '
```

Comment:
This code would result in an error.
```python
>>> "ABC" + (5.5) + "DEF"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: must be str, not float
```
To fix this, you can do `'You cannot upload a file that is more than {} GB of space!'.format(settings.MAX_FILE_SIZE / 
 settings.FILE_SIZE_ONE_GB)`.